TemaConcurrency Sergiu/src/main/java/FestivalStatisticsThread.java

```diff
@@ -0,0 +1,33 @@
+import java.util.LinkedList;
+
+public class FestivalStatisticsThread extends Thread {
+
+    private final FestivalGate gate;
+
+    FestivalStatisticsThread(FestivalGate gate) {
+        this.gate = gate;
+    }
+
+    public void run() {
+        LinkedList<TicketType> queue;
+        while (true) {
+            synchronized (gate) {
+                queue = gate.getQueue();
+            }
+            if (!queue.isEmpty()) {
+                System.out.println(Rainbow.ANSI_CYAN + "\n########## FESTIVAL GATE ENTRANCE MONITORING ##########\n" + Rainbow.ANSI_YELLOW

```

Comment:
well this is actually something original, ++ looks good in the console, check if you can use it in the LOGGER